presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -983,6 +984,11 @@ public SystemSessionProperties(
                         "use a default filter factor for unknown filters in a filter node",
                         featuresConfig.isDefaultFilterFactorEnabled(),
                         false),
+                doubleProperty(
+                        DEFAULT_JOIN_SELECTIVITY_COEFFICIENT,
+                        "use a default join selectivity coefficient factor when column statistics are not available in a join node",
+                        featuresConfig.getDefaultJoinSelectivityCoefficient(),
```

Comment:
check that the value it's set to is within the acceptable range (0 and 1 i would guess)